gql/server.go

```diff
@@ -49,6 +51,13 @@ func (s *Server) Start(ctx context.Context) error {
 
 	// Serve React app
 	mux := http.NewServeMux()
+
+	// Get new cors
+	options := cors.Options{
+		AllowedHeaders: []string{"*"},
+	}
+	c := cors.New(options)
+
```

Comment:
I think we can add CORS request handling in the Start method or in NewServer but probably don't need both.